okta/config.go

```diff
@@ -182,6 +190,99 @@ func oktaSDKClient(c *Config) (client *sdk.Client, err error) {
 	return
 }
 
+// TODU remove err???
```

Comment:
If the method doesn't need to return an error let's remove the error as a return parameter and remove the TODO statement.